ui/package.json

```diff
@@ -10,6 +10,7 @@
     "@material-ui/lab": "^4.0.0-alpha.60",
     "axios": "^1.6.7",
     "babel-eslint": "^10.1.0",
+    "follow-redirects": "^1.15.6",
```

Comment:
Any reason why this has been added? Seems out of place for the PR

Reply:
There was a vulnerability in the dependancy that would cause the node check to fail - if it's best I can remove it and do it as a separate PR?

Reply:
It's fine there! Just wanted to know just in case it was on accident